crates/astria-proto/proto/astria/execution/v1alpha2/execution.proto

```diff
@@ -51,20 +55,19 @@ message CreateBlockRequest {
   google.protobuf.Timestamp timestamp = 3;
 }
 
-// The CommitmentState holds the block at each stage of sequencer commitment level
+// The CommitmentState holds the block at each stage of sequencer commitment
+// level
 //
 // A Valid CommitmentState:
-// - Block numbers are such that head >= soft >= firm.
-// - Head block may either be the same as soft, or have a height 1 higher than soft.
-// - No blocks ever decrease in block number, only head may stay the same and have other changes
-// - The chain defined by head is the canonical chain which always contains soft and firm blocks.
+// - Block numbers are such that soft >= firm.
+// - No blocks ever decrease in block number.
+// - The chain defined by soft is the had of the canonical chain the firm block
+//   must belong to.
```

Comment:
```suggestion
// - The soft block must be a descendant of the current firm block
```

Reply:
I think this verbiage is the opposite of the normal direction of thinking about the chain?